.github/workflows/daily.yml

```diff
@@ -0,0 +1,21 @@
+name: daily
+on:
+  schedule:
+    - cron: '1 * * * *'
+  # can be run manually on https://github.com/cockpit-project/bots/actions
+  workflow_dispatch:
+jobs:
+  npm-update:
```

Comment:
Argh, this should have been a more generic name than "npm-update"; I'll send a follow-up fix.